recipes/radical.saga/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "radical.saga" %}
+{% set version = "0.60.0" %}
+{% set sha256 = "66045355acb70b287f63d439a929ef61c1404d83049640fbd9e0170e8e36f9f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
+  skip: True  # [py3k or win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - apache-libcloud
+    - parse
+    - radical.utils>=0.50.0
```

There has to be a space here to keep some of conda-forge's scripts happy
```suggestion
    - radical.utils >=0.50.0
```

---

Sure, we can do that.  But if you point us to where the condaforge script parsing goes wrong, I can also try to fix it there.  It's kind of cumbersome to remember two different syntaxes for the same thing :-)  Let us know.  If that is known to not be an easy fix, we'll change of course.

---

It's not conda-forge scripts. It's conda-build. According to conda-build spec, there should be a space, but this is not enforced in conda-build everywhere. Some parts of conda-build works without a space, some parts don't. With space works everywhere.

---

If it's the spec, then it's the spec... *sigh*  We change it.